src/vs/workbench/contrib/chat/browser/chatContentParts/chatTodoListWidget.ts

```diff
@@ -190,14 +198,32 @@ export class ChatTodoListWidget extends Disposable {
 			const statusIcon = dom.$('.todo-status-icon.codicon');
 			statusIcon.classList.add(this.getStatusIconClass(todo.status));
 			statusIcon.style.color = this.getStatusIconColor(todo.status);
-
-			const todoContent = dom.$('.todo-content');
+			// Hide decorative icon from screen readers
+			statusIcon.setAttribute('aria-hidden', 'true');
 
 			const titleElement = dom.$('.todo-title');
 			titleElement.textContent = todo.title;
 
+			// Add hidden status text for screen readers
+			const statusText = this.getStatusText(todo.status);
```

Comment:
@meganrogge this seems like a hack to me, is there really no other way of providing ARIA information other than creating a off screen HTML element?

Reply:
@bhavyaus do we need that, given the `todoElement` has an `ariaLabel`? 

https://github.com/microsoft/vscode/blob/c51ec27b9b06377ccd9916ea692d0e175857ae8d/src/vs/workbench/contrib/chat/browser/chatContentParts/chatTodoListWidget.ts#L87-L88